cscs-checks/tools/profiling_and_debugging/notool.py

```diff
@@ -6,12 +6,12 @@
 
 
 @rfm.required_version('>=2.14')
-@rfm.parameterized_test(['C'], ['C++'], ['F90'])
+@rfm.parameterized_test(['C++'], ['F90'])
 class JacobiNoToolHybrid(rfm.RegressionTest):
     def __init__(self, lang):
         super().__init__()
-        self.name = 'jacobi_%s' % lang.replace('+', 'p')
-        self.descr = '%s check' % lang
+        self.name = 'JacobiCheck_%s' % lang.replace('+', 'p')
```

Comment:
I would do the following:

```suggestion
        self.name = '%s_%s' % (type(self).__name__, lang.replace('+', 'p'))
```

This will produce a name such as this: `JacobiNoToolHybrid_Cpp`

Reply:
:ok:

```
[----------] started processing JacobiNoToolHybrid_Cpp (Jacobi (without tool) C++ check)
[ RUN      ] JacobiNoToolHybrid_Cpp on dom:mc using PrgEnv-cray
[ RUN      ] JacobiNoToolHybrid_Cpp on dom:mc using PrgEnv-gnu
...
[----------] finished processing JacobiNoToolHybrid_Cpp (Jacobi (without tool) C++ check)
```